sdk/cosmosdb/cosmos/test/public/functional/computedProperties.spec.ts

```diff
@@ -0,0 +1,278 @@
+    const container: Container = await createContainer(
+      "all computed properties, indexed, include all",
+      allComputedProperties,
+      indexAllComputedPropertiesIncludeAll
+    );
+    const { resources: results } = await container.items
+      .query(selectAllComputedPropertiesQuery)
+      .fetchAll();
+    assert.deepStrictEqual(results, allComputedPropertiesResult);
+  });
+
+  it("all computed properties, not indexed, exclude /*", async function () {
+    const container: Container = await createContainer(
+      "all computed properties, not indexed, exclude all",
+      allComputedProperties,
+      indexDefaultExcludeAll
+    );
+    const { resources: results } = await container.items
+      .query(selectAllComputedPropertiesQuery)
+      .fetchAll();
+    assert.deepStrictEqual(results, allComputedPropertiesResult);
+  });
+
+  it("all computed properties, not indexed, include /*", async function () {
+    const container: Container = await createContainer(
+      "all computed properties, not indexed, include all",
+      allComputedProperties,
+      indexDefaultIncludeAll
+    );
+    const { resources: results } = await container.items
+      .query(selectAllComputedPropertiesQuery)
+      .fetchAll();
+    assert.deepStrictEqual(results, allComputedPropertiesResult);
+  });
+
+  // replace test
+  it("replace computed property for a contianer", async function () {
+    const database = await getTestDatabase("sample database");
+    // create container
+    const { resource: containerdef } = await database.containers.createIfNotExists({
+      id: "replace computed property for a contianer",
+      computedProperties: [lowerName],
+      indexingPolicy: indexDefaultIncludeAll,
+    });
+    const container: Container = database.container(containerdef.id);
+    await createOrUpsertItem(container, sampleItems[0], undefined, false);
+    await createOrUpsertItem(container, sampleItems[1], undefined, false);
+    const { resources: results1 } = await container.items
+      .query(selectAllComputedPropertiesQuery)
+      .fetchAll();
+    const output1 = [
+      {
+        lowerLastName: "andersen",
+      },
+      {
+        lowerLastName: "wakefield",
+      },
+    ];
+
+    assert.deepStrictEqual(results1, output1);
+    containerdef.computedProperties = [parentsFullName];
+    await container.replace(containerdef);
+    const { resources: results2 } = await container.items
+      .query(selectAllComputedPropertiesQuery)
+      .fetchAll();
+    const output2 = [
+      {
+        parentsFullName: "Thomas Andersen & Mary Kay Andersen",
+      },
+      {},
+    ];
+    assert.deepStrictEqual(results2, output2);
+  });
+
+  it("delete computed property for a contianer", async function () {
+    const database = await getTestDatabase("sample database");
+    // create container
+    const { resource: containerdef } = await database.containers.createIfNotExists({
+      id: "replace computed property for a contianer",
```

Comment:
Id of the container can be changed to "delete computed property for a container" for readability.